### DISCUSSION TITLE
Feature request: Leaving penalty

### DISCUSSION THREAD

**Discussion by ShiroWork-s** [redeclipse/base#1266] [Ideas and Feature Requests]
Ik it might be annoying on some games, Apex, Rocket League, 
but it really does work. The idea of getting locked from playing a game for around five or ten minutes after leaving mid-game works in many ways.
Its prevents toxcity,
And ive heard of people saying, "Oh, let's say our players needed to go do something, or wanted to eat, that wouldnt be nice to them!" Thats when you think, eating or doing something takes usually around ten minutes anyways, so it would only really hurt the people who want to hop out to create a new game because they are losing the current one.

**Comment by Calinou** [comment]
This can't work on a cross-server basis without some kind of cooperation from the master server. Either way, I don't see this happening in Red Eclipse given the focus on casual gameplay (where leaving penalties are uncommon).

**Reply by ShiroWork-s** [reply]
I see,
I'm all for the casual gameplay, yet if Red Eclipse ever goes into the subject of tournaments and ranked matches, keep it in mind. :D
